Engine/Game/SceneFactory.cpp

```
@@ -0,0 +1,15 @@
+#include "SceneFactory.h"
+
+#include "GameScene.h"
+
+std::unique_ptr<BaseScene> SceneFactory::Create(const std::string& name) {
```

cppcheck (2.10-2)

[style] The function 'Create' is never used.

(unusedFunction)